### PR TITLE
feat: (Replat 4657) Web - Respect disable Save/ Share flags on an article

### PR DESCRIPTION
**Pull request by jmars**

### CODE REVIEW

**Comment by nehasri89** [comment]
This PR being branched from placement bar is not really up for review I think, is good for dev but can take it off from PR review

**Comment by ValGeorgiev** [comment]
Save and Share bar anchoring is merged last week, could we create a PR for this from the master branch?

**Comment by aronshamash** [comment]
@jmars Can you merge in master and then raise PR

File: packages/article-skeleton/src/article-skeleton.web.js [hunk 1]
```diff
@@ -55,10 +55,12 @@ class ArticleSkeleton extends Component {
   }
 
   handleScroll() {
-    const offsetTop = this.sticky.getBoundingClientRect().top;
-    const isSticky = isStickyAllowed(breakpoints.huge) && offsetTop <= 1;
+    if (this.stick) {
```
Comment (ValGeorgiev) [inline packages/article-skeleton/src/article-skeleton.web.js]: typo, it should be `this.sticky`

File: packages/article-skeleton/src/article-skeleton.web.js [hunk 2]
```diff
@@ -129,24 +133,26 @@ class ArticleSkeleton extends Component {
...
+                                  this.sticky = el;
+                                }}
+                              >
+                                <SaveAndShareBar
+                                  articleId={articleId}
+                                  articleHeadline={headline}
+                                  articleUrl={url}
+                                  onCopyLink={() => {}}
+                                  onSaveToMyArticles={() => {}}
+                                  onShareOnEmail={() => {}}
```
Comment (ValGeorgiev) [inline packages/article-skeleton/src/article-skeleton.web.js]: i see you access `savingEnable` and `sharingEnable` in the SaveAndShareBar, but i cannot see them here

File: packages/save-and-share-bar/src/save-and-share-bar.js [hunk 3]
```diff
@@ -30,68 +30,74 @@ class SaveAndShareBar extends Component {
...
-            onPress={onSaveToMyArticles}
-          >
-            <IconSaveBookmark
-              fillColour={styles.svgIcon.save.fillColour}
-              strokeColour="currentColor"
-              title="Save to My Articles"
-            />
-          </BarItem>
-        </View>
+        {sharingEnabled && (
```
Comment (nehasri89) [inline packages/save-and-share-bar/src/save-and-share-bar.js]: think `sharingEnable` should work for the whole Save and Share?

File: packages/article-magazine-comment/__tests__/web/__snapshots__/article.web.test.js.snap [hunk 4]
```diff
@@ -185,6 +185,8 @@ exports[`3. an article with no headline falls back to use shortheadline 1`] = `
                   },
                   "publicationName": "TIMES",
                   "publishedTime": "2015-03-23T19:39:39.000Z",
+                  "savingEnabled": true,
```
Comment (nehasri89) [inline packages/article-magazine-comment/__tests__/web/__snapshots__/article.web.test.js.snap]: should have test for when they are false as well

**Comment by parkjh** [comment]
@jmars you got conflicts... :(

**Pull request merged**